x/bank/internal/types/expected_keepers.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/exported"
+)
+
+// AccountKeeper defines the account contract that must be fulfilled when
+// creating a x/bank keeper.
+type AccountKeeper interface {
+	NewAccountWithAddress(ctx sdk.Context, addr sdk.AccAddress) exported.Account
+
+	GetAccount(ctx sdk.Context, addr sdk.AccAddress) exported.Account
```

Comment:
@alessio It's methods like this in interfaces which cause the need for `x/{module}/exported`. i.e. I'd like to define some contract w/o importing the originating module (in this case `x/auth`'s `alias.go`). This contract contains methods which operate on and/or return types that are unique to that module (in this case `Account`). In order to use this contract successfully, I have to be able to import said types w/o having import cycle hell, hence `x/{module}/exported`.